parsons/bluelink/person.py

```diff
@@ -0,0 +1,242 @@
+import logging
+import json
+
+logger = logging.getLogger(__name__)
+
+
+class Person(object):
+    """
+    Instantiate Person Class.
+    Used for to upserting via Bluelink connector.
+    See: https://bluelinkdata.github.io/docs/BluelinkApiGuide#person-object
+
+    `Args:`
+        identifiers: list[Identifier]
+            A list of Identifier objects.
+            A Person must have at least 1 identifier
+        given_name: str
+            First name / given name
+        family_name: str
+            Last name / family name
+        emails: list[Email]
+            A list of Email objects representing email addresses.
```

Comment:
Can you update all the args descriptions to be consistent with punctuation? Either end all in `.` or none

Reply:
I'll make it a task to read up on technical documentation style. I'm not sure what the best practice to do in certain situations, like when ending a sentence with a quote character, where the quotes are being used to indicate a literal. Other than those weird situations, I changed it so that every sentence ends in a period.